dist/rudder-sdk-js/index.d.ts

```diff
@@ -399,7 +410,9 @@ declare module "rudder-sdk-js" {
   /**
    * To get anonymousId set in the SDK
    */
-  function getAnonymousId(): string;
+  function getAnonymousId(
+    anonymousIdOptions: undefined | anonymousIdOptions
```

Comment:
The right way to make it an optional argument.
```suggestion
    anonymousIdOptions?: anonymousIdOptions
```

Reply:
Done